multiworld/world_communicator.py

```diff
@@ -102,7 +102,18 @@ async def _wait_work(self, work: Work, world_name: str) -> None:
     async def send(
         self, tensor: Tensor, dst: int, world_name: str = DEFAULT_WORLD_NAME
     ) -> None:
-        """Send a tensor to a destination in a world."""
+        """
+        Send a tensor to a destination in a world.
+
+        Args:
+            tensor: Tensor to be sent.
+            dst: Destination rank from the world.
+            world_name: Name of the world.
+
+        Raises:
+            BrokenWorldException: An error that occurs when
+            the world is broken due to worker, node or network failure.
```

Comment:
```suggestion
                the world is broken due to worker, node or network failure.
```
please adjust indentation. add 4 more spaces.